modules/osd-monitoring-support-considerations.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-configuring-the-monitoring-stack.adoc
+
+[id="support-considerations_{context}"]
+= Support considerations for monitoring
+
+The following modifications are explicitly not supported:
+
+* *Creating additional `ServiceMonitor`, `PodMonitor`, and `PrometheusRule` objects in the `openshift-&#42;`, and `kube-&#42;` projects.*
```

Comment:
I believe all or most of this should stay. I think the note about the Alertsmanager should be removed since alerts are not supported at all.

@sam-nguyen7  @sichvoge What parts of this topic should be deleted for OSD and ROSA?

Reply:
Cluster Admins could do this in theory. I would leave the explicit warning

Reply:
Keeping it.